[All Hosts] (webviews) emphasize webview terms instead of brower terms (#4018)
OfficeDev/office-js-docs-pr: docs/develop/auth-with-office-dialog-api.md

Office Add-ins run in an iframe when opened in Office on the web. Many identity authorities, also called Secure Token Services (STS), prevent their sign-in page from opening in an iframe. These include Google, Facebook, and services protected by the Microsoft identity platform (formerly Azure AD V 2.0) such as a Microsoft account, a Microsoft 365 Education or work account, or other common account. Also security features implemented in the webview when Office Add-ins run in Office on Windows, or Office on Mac can prevent sign-in pages from working correctly.

> For authorization to work correctly, the sign-in page must open in a separate browser or webview control instance. This is why Office provides the [Office dialog API](dialog-api-in-office-add-ins.md), specifically the [displayDialogAsync](/javascript/api/office/office.ui) method. 

Reviewer: ```suggestion
For authorization to work correctly, the sign-in page must open in a separate browser or webview control instance. This is why Office provides the [Office dialog API](dialog-api-in-office-add-ins.md), specifically the [displayDialogAsync](/javascript/api/office/office.ui) method.
```

> [!NOTE]
>

> > - This article assumes that you are familiar with [Use the Office dialog API in your Office Add-ins](dialog-api-in-office-add-ins.md).

Reviewer: ```suggestion
> - This article assumes that you're familiar with [Use the Office dialog API in your Office Add-ins](dialog-api-in-office-add-ins.md).
```
